vyxalbot2/commands.py

```diff
@@ -53,8 +53,8 @@
     (
         r"(much |very |super |ultra |extremely )*(good|great|excellent|gaming) bot!*",
     ): "!good-bot",
-    (r"(hello|howdy|mornin['g]|evenin['g])( y'?all)?",): "hello",
-    (r"((good)?bye|see ya\!?|'night|goodnight)( y'?all)?",): "goodbye",
+    (r"(hello|howdy|(good )?mornin['g]|(good )?evenin['g])( y'?all)?",): "hello",
+    (r"((good)?bye|see ya\!?|'night|(good|night )night)( y'?all)?",): "goodbye",
```

Comment:
```suggestion
    (r"((good)?bye|see ya\!?|'night|(good ?|night )?night)( y'?all)?",): "goodbye",
```
To allow `good night` with a space in between, as well as just `night`